src/sso/sso.spec.ts

```diff
@@ -112,6 +112,42 @@ describe('SSO', () => {
           expect(url).toMatchSnapshot();
         });
       });
+
+      describe('with domain_hint', () => {
```

Comment:
nit: Can we make this match the property name in the public interface?

```suggestion
      describe('with domainHint', () => {
```